x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -753,17 +753,17 @@ NOTE: in this context, `body.*` will be the result of all the previous transform
 [float]
 ==== `response.split[].keep_parent`
 
-If set to true, the fields from the parent document (at the same level as `target`) will be kept. Otherwise a new document will be created using `target` as the root. Default: `false`.
+If set to true, the fields from the parent document (at the same level as `target`) will be kept. Otherwise, a new document will be created using `target` as the root. Default: `false`.
 
 [float]
 ==== `response.split[].delimiter`
 
-Required if using split type of `string`.  This is the sub string used to split the string.  For example if `delimiter` was "\n" and the string was "line 1\nline 2", then the split would result in "line 1" and "line 2".
+Required if using the split type of `string`.  This is the substring used to split the string.  For example, if `delimiter` is "\n" and the string was "line 1\nline 2", then the split would result in "line 1" and "line 2".
```

Comment:
Two choices.

For formally correct English using subjunctive:
```suggestion
Required if using the split type of `string`.  This is the substring used to split the string.  For example, if `delimiter` were "\n" and the string were "line 1\nline 2", then the split would result in "line 1" and "line 2".
```
For less correct but more modern English:
```suggestion
Required if using the split type of `string`.  This is the substring used to split the string.  For example, if `delimiter` is "\n" and the string is "line 1\nline 2", then the split will result in "line 1" and "line 2".
```
(note agreement in mood in both cases)

Reply:
Ah, okay. Going with the 2nd one based on the other lines because they are modern English.